apps/win32/nuget/Microsoft.FluentUI.FluentTesterWin32.nuspec

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+  <metadata>
```

Comment:
You should include this in the metadata:

```xml
  <metadata>
    ...
    <repository type="git"
      url="https://github.com/microsoft/fluentui-react-native.git"
      commit="$CommitId$" />
  </metadata>
```

Then add `-properties CommitId=$(Build.SourceVersion)` to the pack commandline.  Then it is easier to track where the nuget comes from.

Reply:
Changed in newest iteration - thanks for the advice